skimage/io/_plugins/freeimage_plugin.py

```diff
@@ -763,5 +763,26 @@ def imsave(filename, img):
     ----------
       filename : file name
       img : image to be saved as nd array
+
+    Other parameters
+    ----------------
+    kwargs : keywords
+        When saving as JPEG, supports the ``quality`` keyword argument which is
+        an integer with values in [1, 100]
```

Comment:
I have the feeling that the docstring does not inform properly on the behavior of this option. quality=76 or 100 will return the exact same picture. I don't know if this is standards for jpeg, but I never had this intuition with gimp of whatever.